### ISSUE TITLE
Docker image volume isn't being mounted properly

### ISSUE THREAD

**Issue by ntwrkguru** [Juniper/py-junos-eznc#612]
```
$ docker run -it juniper/pyez
/scripts # ls
/scripts #
```

Manually mounting the volume does work, however:

```
$ docker run -it --rm -v $PWD:/scripts juniper/pyez sh
/scripts # ls
README.md   errors.txt  fetcher.py  hosts.csv
/scripts #
```

**Comment by ydnath** [comment]
@dgarros Can you please check this?

**Comment by stacywsmith** [comment]
Addressed by PR #673.
